modules/ducktests/tests/ignitetest/services/ignite.py

```diff
@@ -154,3 +213,14 @@ def pids(self, node):
             return pid_arr
         except (RemoteCommandError, ValueError):
             return []
+
+    def ssh_output_all(self, cmd: str):
```

Comment:
execute on every node?
why it's located at ignite.py, not at ignite_aware?

Reply:
fixed